fusor-ember-cli/app/components/base-f.js

```diff
@@ -3,12 +3,14 @@ import Ember from 'ember';
 export default Ember.Component.extend({
 
   didInsertElement() {
-    return Ember.$('[data-toggle="popover"]').popover({html: false,
-                                                       trigger: 'focus hover',
-                                                       title: this.get('label'),
-                                                       placement: 'right',
-                                                       content: this.get('helpText')
-                                                      });
+    const $component = this.$();
```

Comment:
I know it's just a var name, but when something starts with $ I think it's part of some library.  I'm guessing you wanted to indicate it's a jquery Object?